src/spatial_ref/srs.rs

```diff
@@ -506,6 +506,22 @@ impl SpatialRef {
         res
     }
 
+    pub fn to_projjson(&self) -> Result<String> {
+        let mut c_projjsonstr = ptr::null_mut();
+        let options = ptr::null();
+        let rv = unsafe { gdal_sys::OSRExportToPROJJSON(self.0, &mut c_projjsonstr, options) };
+        let res = if rv != OGRErr::OGRERR_NONE {
+            Err(GdalError::OgrError {
+                err: rv,
+                method_name: "OSRExportToPROJJSON",
+            })
+        } else {
+            Ok(_string(c_projjsonstr))
+        };
+        unsafe { gdal_sys::VSIFree(c_projjsonstr.cast::<std::ffi::c_void>()) };
```

Comment:
I don't even know any more which one of `.cast()`, `.cast<c_void>()` or `as _` is preferred :smiley:. At least this is consistent with the rest of the code.

Reply:
Yeah, it's mostly a copy/paste... The entire lib could do with some helper functions/macros to streamline stuff like this.